src/taskManager/UserInterface.java

```diff
@@ -0,0 +1,114 @@
+package taskManager;
+
+import java.util.Scanner;
+
+public class UserInterface {
+    Scanner sc = new Scanner(System.in);
+    TaskList taskList = new TaskList();
+
+    public void startUI(){
+        while(true){
+            printCommandOptions();
+        }
+    }
+
+    public void printCommandOptions() {
+        System.out.println("Select command you would like to implement from following:");
+        System.out.println("1 - add Task");
+        System.out.println("2 - edit Task");
+        System.out.println("3 - remove Task");
+        System.out.println("4 - print Tasks List");
+        System.out.println("5 - change task status");
+        System.out.println("exit - exit the application");
+        handleInput();
+    }
```

Comment:
Methods should do one thing. Currently, this `printCommandOptions` method does not just print messages but also handles user input.

```suggestion
    public void startUI(){
        while(true){
            printCommandOptions();
            handleInput();
        }
    }

    public void printCommandOptions() {
        System.out.println("Select command you would like to implement from following:");
        System.out.println("1 - add Task");
        System.out.println("2 - edit Task");
        System.out.println("3 - remove Task");
        System.out.println("4 - print Tasks List");
        System.out.println("5 - change task status");
        System.out.println("exit - exit the application");
    }
```